lib/game.rb

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env ruby
+require_relative '../lib/player.rb'
+class Game
+  def create_player(name, token, move_count)
+    player = Player.new(name, token, move_count)
+    player
+  end
```

Comment:
Why that line? 
Why not simply instantiate it in `bin/main.rb`?
There is apparently no reason for that method in that class